src/algorithms/digi/TOFPulseDigitization.cc

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Souvik Paul, Chun Yuen Tsang, Prithwish Tribedy
+// Special Acknowledgement: Kolja Kauder
+//
+// Convert ADC pulses from TOFPulseGeneration into ADC and TDC values
+
+#include <math.h>
+#include <podio/RelationRange.h>
+#include <stdlib.h>
+#include <algorithm>
+#include <gsl/pointers>
+#include <limits>
+#include <vector>
+
+#include "TOFPulseDigitization.h"
+#include "algorithms/digi/TOFHitDigiConfig.h"
+
+
+namespace eicrecon {
+
+void TOFPulseDigitization::process(const TOFPulseDigitization::Input& input,
+                                   const TOFPulseDigitization::Output& output) const {
+  const auto [simhits] = input;
+  auto [rawhits] = output;
+
+  double thres = m_cfg.t_thres;
+  // double Vm=-0.05;
+  //  SP noted that max dE experienced by LGAD should be 0.8 keV
```

Comment:
Minor point of code hygiene: since the code here is agnostic to the value of the parameter, I would suggest moving this comment to the plugin where you specify what the value of `Vm` is.

Not to mention (following some of @simonge's suggestions) this code might be reused potentially other electronics like MAPS or timepix, which means this value might be different from -0.05.

Reply:
I added a comment to describe what Vm does. I also added units to TOFHitDigiConfig.h